charts/postgresql/templates/_helpers.yaml

```diff
@@ -0,0 +1,7 @@
+{{- define "astronomer_bootstrap_url" -}}
+{{- if .Values.verify_ssl }}
+postgres://{{ .Values.postgresql.postgresqlUsername }}:{{ .Values.postgresql.postgresqlPassword }}@{{ .Release.Name }}-postgresql.{{ .Release.Namespace }}.svc.cluster.local:{{.Values.postgresql.servicePort}}?sslmode=verify-ca"
```

Comment:
@andriisoldatenko what other modes are available? Should we support a custom `mode` option, rather than the boolean?